src/DotLiquid/DotLiquid.csproj

```diff
@@ -41,6 +41,10 @@
     </None>
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+    <PackageReference Include="System.Text.Json" Version="4.7.2" />
```

Comment:
Create a separate assembly project to add new dependency to Json so client can opt-in. Move the new filter(s) depending on Json there.

Reply:
The new json filter will be much welcome by users of Logic App. However we need to introduce this via a separate assembly so we don't force it on other consumers.
Technically you may also need to generate a separate nuget package so you can declare the new nuget dependency separately as well.

Reply:
I'm not familiar with NuGet policies, but will we simply be able to add a new target in appveyor for a new nuspec package and upload with the existing API key? Or will @tgjones need to do something first on NuGet to register the name / allow publish via API?

Reply:
I've removed the dependency and the Json filter from the PR.